src/_layouts/docs.html

```diff
@@ -51,22 +51,26 @@ <h3 class="sidebar-sectionTitle">{{ section.title | escape }}</h3>
 
   </nav>
   <article class="docsContent">
-    {% if page.hideEditButton != true %}
-    <a class="editOnGitHub u-hideOnMobile" 
-       href="https://github.com/tilt-dev/tilt.build/tree/master/docs/{{page.name}}">
-      {% svg assets/svg/github.svg width=16 height=16 %}
-
-      <span class="editOnGitHub-text">
-        Edit on GitHub
-      </span>
-    </a>
-    {% endif %}
-    
     <h1>{{ page.title | escape }}</h1>
     
     {{ content }}
 
+    <div class="docsFooter">
     {% include helpful_form.html %}
+    <a id="scrollToTop" href="#">
```

Comment:
i noticed that the footer links are _slightly_ misaligned. if you add `display: block; float: left;` to this link, it should align the items!

![Screen Shot 2021-08-17 at 11 19 11 AM](https://user-images.githubusercontent.com/23283986/129755788-4d4de1ba-1639-495b-9f17-0219b19867d5.png)

Reply:
😓 (fixed in 6c98e1f)